azurecompute-arm/pom.xml

```diff
@@ -135,6 +135,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.4</version>
+    </dependency>
```

Comment:
This is added only to join a string. Remove the dependency and use Guava's `Joiner`.